features/git-ship/current_branch/commit_message_containing_double_quote.feature

```diff
@@ -0,0 +1,39 @@
+Feature: git ship: shipping the current feature branch
+
+  As a developer entering a commit message that contains a double quote
+  I want it to still work as expected
+  So shipping is a robust process.
+
+  Background:
+    Given I have a feature branch named "feature"
+    And the following commit exists in my repository
+      | BRANCH  | LOCATION | FILE NAME    | FILE CONTENT    |
+      | feature | remote   | feature_file | feature content |
+    And I am on the "feature" branch
+    When I run `git ship -m 'feature "done"'`
```

Comment:
How about
> When I run `git ship -m 'commit message containing "double-quotes"'`

the *feature* doesn't really have quotes, it's the message that does